bridge/README.md

```diff
@@ -1,120 +1,17 @@
 # AliceNet/Bridge
```

Comment:
Just a quick note: I think this is fine for the time being, but we should move `package.json` to the root, at which point we can merge anything needed from this README to the root README as well.